packages/oracle/src/_internal/connection-options.ts

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved
+
+import type { PickByType } from '@sequelize/utils';
+import { getSynchronizedTypeKeys } from '@sequelize/utils';
+import type { OracleConnectionOptions } from '../connection-manager.js';
+
+type StringConnectionOptions = PickByType<OracleConnectionOptions, string>;
+
+const STRING_CONNECTION_OPTION_MAP = {
+  configDir: undefined,
+  connectionIdPrefix: undefined,
+  connectString: undefined,
+  connectionString: undefined,
```

Comment:
These two do the same thing, so we should only accept one of them. I recommend to omit `connectString` from `OracleConnectionOptions`